velox/type/Type.h

```diff
@@ -507,6 +540,61 @@ class TypeBase : public Type {
   }
 };
 
+using ShortDecimalType = DecimalType<TypeKind::SHORT_DECIMAL>;
+using LongDecimalType = DecimalType<TypeKind::LONG_DECIMAL>;
+
+/// This class represents the fixed-point numbers.
+/// The parameter "precision" represents the number of digits the
```

Comment:
Maybe clarify there will be as multiple Decimal types in the application differing by precision and scale.